src/fstests/fstests.jinja2

```diff
@@ -3,9 +3,16 @@
 
 {%- extends 'base/python.jinja2' %}
 
+{%- block python_globals %}
+{{ super() }}
+xfstests_bld_path = '{{ xfstests_bld_path }}'
```

Comment:
This could be passed to the constructor instead of a global variable with `{% block python_job_constr %}`.